src/components/sponsors_tier.css

```diff
@@ -0,0 +1,112 @@
+.contentDataPlaceholder {
```

Comment:
Loved the animation. But preferably not to use separate css. Could try using Preline animation for the carousel. I had a quick look and it looks pretty simple. You can even do the infinite-loop for automatic scrolling as well! https://preline.co/plugins/html/carousel.html